libs/client-sdk/tests/client_c_bindings_test.cpp

```diff
@@ -18,5 +18,9 @@ TEST(ClientBindings, MinimalInstantiation) {
 
     LDClientSDK sdk = LDClientSDK_New(config, context);
 
+    char const* version = LDClientSDK_Version();
+    ASSERT_TRUE(version);
+    ASSERT_STREQ(version, "0.1.0");  // {x-release-please-version}
```

Comment:
Need to add these files to release-please-config.json as well.

It only looks at specific files to try to update them.
```
      "extra-files": [
        "include/launchdarkly/client_side/client.hpp"
      ]
```

Reply:
Ah, missed that.